parsers/ENTSOE.py

```diff
@@ -810,9 +810,12 @@ def parse_production(
         for entry in timeseries.find_all("point"):
             quantity = float(entry.find_all("quantity")[0].contents[0])
             position = int(entry.find_all("position")[0].contents[0])
+            is_production = (
```

Comment:
Could you add a comment describing what this is for here ?

Reply:
Sure thing!